tests/bgp/test_ipv6_bgp_scale.py

```diff
@@ -37,7 +37,7 @@
 IPV6_KEY = "ipv6"
 MAX_BGP_SESSIONS_DOWN_COUNT = 0
 MAX_DOWNTIME = 10  # seconds
-MAX_DOWNTIME_ONE_PORT_FLAPPING = 30  # seconds
+MAX_DOWNTIME_PORT_FLAPPING = 30  # seconds
 MAX_DOWNTIME_UNISOLATION = 300  # seconds
```

Comment:
should we rename the unisolation time to "MAX_DOWNTIME_ALL_PORT_FLAPPING" now? and does it need to still be different?

since it is a linear, should we make the max downtime calculated from something?